pylint_silent/__init__.py

```diff
@@ -162,7 +168,7 @@ def statistics(py_filenames: List[str]) -> None:
 
                     # 'comment' may disable several messages:
                     # "# pylint: disable=too-many-branches,too-many-statements"
-                    messages = comment[comment.rfind("=") + 1:].split(",")
+                    messages = comment[comment.rfind("=") + 1:].split(';')[0].split(",")
```

Comment:
When signature is used, we should only count comment lines with signature.

When signature is not used, I would like to count only lines without signature, but that might confuse users.
Maybe in the future, after making `--signature` the default, I will add a `--no-signature` option that will implement this logic. It could also restrict `reset` to lines without signatures.

Reply:
add signature logic